examples/release_highlights/plot_release_highlights_1_6_0.py

```diff
@@ -0,0 +1,212 @@
+# ruff: noqa
+"""
+=======================================
+Release Highlights for scikit-learn 1.6
+=======================================
+
+.. currentmodule:: sklearn
+
+We are pleased to announce the release of scikit-learn 1.6! Many bug fixes
+and improvements were added, as well as some key new features. Below we
+detail the highlights of this release. **For an exhaustive list of
+all the changes**, please refer to the :ref:`release notes <release_notes_1_6>`.
+
+To install the latest version (with pip)::
+
+    pip install --upgrade scikit-learn
+
+or with conda::
+
+    conda install -c conda-forge scikit-learn
+
+"""
+
+# %%
+# FrozenEstimator: Freezing an estimator
+# --------------------------------------
+#
+# This meta-estimator allows you to take an estimator and freeze its fit method, meaning
+# that calling `fit` does not perform any operations; also, `fit_predict` and
+# `fit_transform` call `predict` and `transform` respectively without calling `fit`. The
+# original estimator's other methods and properties are left unchanged. An interesting
+# use case for this is to use a pre-fitted model as a transformer step in a pipeline
+# or to pass a pre-fitted model to some of the meta-estimators. Here's a short example:
+
+import time
+from sklearn.datasets import make_classification
+from sklearn.frozen import FrozenEstimator
+from sklearn.linear_model import SGDClassifier
+from sklearn.model_selection import FixedThresholdClassifier
+
+X, y = make_classification(n_samples=1000, random_state=0)
+
+start = time.time()
+classifier = SGDClassifier().fit(X, y)
+print(f"Fitting the classifier took {(time.time() - start) * 1_000:.2f} milliseconds")
+
+start = time.time()
+threshold_classifier = FixedThresholdClassifier(
+    estimator=FrozenEstimator(classifier), threshold=0.9
+)
```

Comment:
```suggestion
).fit(X, y)
```

Reply:
haha that's how you make good benchmarks ! 😄